.github/workflows/codeql.yml

```diff
@@ -0,0 +1,53 @@
+name: "CodeQL"
+
```

Comment:
we should add the permissions thing I added in the other workflows.

Reply:
What we have is more restrictive already in most cases and we need to be able to write to security events.

Reply:
THe OpenSSF Scorecard I think wants a default premission you then can overwrite.